spoti-vote/src/components/Home.jsx

```diff
@@ -1,47 +0,0 @@
-import React, {Component} from 'react';
```

Comment:
whole deletion should be reverted since Home.jsx is for future usage

Reply:
I dont know why its there, since i have not removed it in my project, and its still here